### DISCUSSION TITLE
Multiple schedulers

### DISCUSSION THREAD

**Discussion by jswiniarski** [quartznet/quartznet#1104] [Q&A]
Hi,

I have a setup of two web applications A and B. Each one has its own DI container configured and its own Quartz scheduler configured with unique id and in-memory store. I also have a functional test project that runs both services. I'm getting errors indicating that scheduler from service A want to instantiate a job that is configured for service B and it fails because Job class is registered in container from service B. Right now my wild guess is that although there are two separate containers and two separate schedulers and two separate hosted services, the RAM job store is a singleton.

Is that true or there is maybe another explanation for my issue?

**Comment by lahma** [comment]
Scheduler name is used to bind to global in-process registry. You could try making sure that the container configurations both set a different name for the scheduler and check if that helps.

**Reply by jswiniarski** [reply]
Yeap, that works. Thank you.
